tracer/test/Datadog.Trace.Tests/Configuration/TracerSettingsTests.cs

```diff
@@ -1382,6 +1382,125 @@ public void LogsInjectionEnabled(string value, bool expected)
+        public void OtelMetricExportTimeoutCustomValues(string value, int expected)
+        {
+            var source = CreateConfigurationSource((ConfigurationKeys.OpenTelemetry.MetricExportTimeout, value));
+            var settings = new TracerSettings(source);
+
+            settings.OtelMetricExportTimeout.Should().Be(expected);
+        }
+
+        [Theory]
+        [InlineData(null, null, OtlpProtocol.HttpProtobuf)]
+        [InlineData("invalid", null, OtlpProtocol.HttpProtobuf)]
+        [InlineData("grpc", null, OtlpProtocol.Grpc)]
+        [InlineData("grpc", "http/protobuf", OtlpProtocol.Grpc)]
+        [InlineData(null, "http/json", OtlpProtocol.HttpJson)]
+        public void OtlpProtocolFallbacks(string metricsProtocol, string generalProtocol, object expected)
+        {
+            var source = CreateConfigurationSource(
+                (ConfigurationKeys.OpenTelemetry.ExporterOtlpMetricsProtocol, metricsProtocol),
+                (ConfigurationKeys.OpenTelemetry.ExporterOtlpProtocol, generalProtocol));
+            var settings = new TracerSettings(source);
+
+            settings.OtlpMetricsProtocol.Should().Be((OtlpProtocol)expected);
+        }
+
+        [Theory]
+        [InlineData("v1", SchemaVersion.V1)]
+        [InlineData("V1", SchemaVersion.V1)]
+        [InlineData("", SchemaVersion.V0)]
+        [InlineData(null, SchemaVersion.V0)]
+        [InlineData("v1 ", SchemaVersion.V0)]
+        public void MetadatsaSchemaVersion(string value, object expected)
+        {
+            var source = CreateConfigurationSource((ConfigurationKeys.MetadataSchemaVersion, value));
+            var settings = new TracerSettings(source);
+
+            settings.MetadataSchemaVersion.Should().Be((SchemaVersion)expected);
+        }
+
+        [Theory]
+        [InlineData("http://custom:4318/v1/metrics", null, "http://custom:4318/v1/metrics")]
+        [InlineData("http://custom:4318/v1/metrics", "http://fallback:4318", "http://custom:4318/v1/metrics")]
+        [InlineData(null, "http://fallback:8080", "http://fallback:8080/v1/metrics")]  // HTTP fallback gets v1/metrics
+        [InlineData(null, null, "http://localhost:4318/v1/metrics")]  // Default
+        public void OtlpEndpointFallbacks(string metricsEndpoint, string generalEndpoint, string expected)
+        {
+            var source = CreateConfigurationSource(
+                (ConfigurationKeys.OpenTelemetry.ExporterOtlpEndpoint, generalEndpoint),
+                (ConfigurationKeys.OpenTelemetry.ExporterOtlpMetricsEndpoint, metricsEndpoint));
+            var settings = new TracerSettings(source);
+
+            settings.OtlpMetricsEndpoint.Should().Be(expected);
+        }
+
+        [Theory]
+        [InlineData("DELTA", OtlpTemporality.Delta)]
+        [InlineData("cumulative", OtlpTemporality.Cumulative)]
+        [InlineData("loWmemOry", OtlpTemporality.LowMemory)]
+        [InlineData(null, OtlpTemporality.Delta)]
```

Comment:
I think we could also use a bogus value like this
```suggestion
        [InlineData(null, OtlpTemporality.Delta)]
        [InlineData("bad-value", OtlpTemporality.Delta)]
```